guide/ops/locations/more-clouds.md

```diff
@@ -172,4 +172,86 @@ For example, for dev/demo purposes an "everything" network could be created that
 || Source IP Ranges            || 0.0.0.0/0                   |
 || Allowed protocols and ports || tcp:0-65535 and udp:0-65535 |
 
-
+## IBM SoftLayer
+
+### VLAN Selection
+
+For some applications it is important that All VMs use the same VLAN. On
+SoftLayer this can be done by setting the `SoftLayerTemplateOptions`
+fields `primaryNetworkComponentNetworkVlanId` and
+`primaryBackendNetworkComponentNetworkVlanId` when specifying the
+location being used in the blueprint, as follows:
+
+```YAML
+location:
+  jclouds:softlayer:
+    region: ams01
+    templateOptions:
+      # Enter your preferred network IDs
+      primaryNetworkComponentNetworkVlanId: 1153481
+      primaryBackendNetworkComponentNetworkVlanId: 1153483
+```
+
+This will only work if you already have a VM configured using the same
+VLAN, and can copy the IDs from the SoftLayer console UI.
+
+If you are deploying a new application, then you can customise the
+location with ` `JcloudsLocationCustomizer` class, and we provide an
+implementation that ensures all machines in a particular scope will
+use the same VLAN. By default, the scope used is the application ID
+for the blueprint being deployed, but this can be changed if you need
+multiple applications to share the same VLAN.
+
+Use the `SoftLayerSameVlanLocationCustomizer` in your blueprint by
```

Comment:
```
To use this method, we tell brooklyn to use `SoftLayerSameVlanLocationCustomizer` as a location customizer.  
This can be done on a location as follows:
```

(put location first, then)

```
It is also possible with many blueprints to specify this as one of the `provisioning.properties` on an *application*:
```

(app blueprint illustration)